<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WgtTileSetCollision</class>
 <widget class="QWidget" name="WgtTileSetCollision">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>156</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Build Settings</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frmBorder">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="btnUpward">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDownward">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDelete">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnSetFilter">
          <property name="text">
           <string>Set Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtFilter"/>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnSetMat">
          <property name="text">
           <string>Set Material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMatName"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnMatColor">
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="chkIsSensor">
        <property name="text">
         <string>Is Sensor</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Paint Tools</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Animation</string>
   </property>
   <property name="toolTip">
    <string>Removes the Animation from the TileSet</string>
   </property>
  </action>
  <action name="actionUpward">
   <property name="icon">
    <iconset resource="../../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-up.png</normaloff>:/icons16x16/generic-up.png</iconset>
   </property>
   <property name="text">
    <string>Order Upward</string>
   </property>
   <property name="toolTip">
    <string>Sorts this animation upward in the list</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDownward">
   <property name="icon">
    <iconset resource="../../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-down.png</normaloff>:/icons16x16/generic-down.png</iconset>
   </property>
   <property name="text">
    <string>Order Doward</string>
   </property>
   <property name="toolTip">
    <string>Sorts this animation downward in the list</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../data/HyEditor.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
